lib/calendar.ts

```diff
@@ -1,5 +1,5 @@
 import { DEBUG } from './debug';
-import { ES } from './ecmascript';
+import * as ES from './ecmascript';
```

Comment:
At first I was concerned that using `import *` would break tree-shaking, but according to https://stackoverflow.com/a/64569600 this is fine. 

But... I'd still like us to verify that TS's transpilation doesn't break webpack's ability to tree-shake this. Any idea how to test this without doing lots of probably-unnecessary work to import each function separately in each non-ecmascript.ts file?  I guess we could just look at how TS emits JS for this case, specifically whether the underlying JS is still `import *` or if it's transpiled into something that will break tree shaking.

Reply:
The underlying JS that tsc emits is:

```
import * as ES from './ecmascript';
```

(from `./tsc-out/calendar.js` after running `npm run build`).

At that point, so long as Webpack says they support tree-shaking of this pattern we should be good?

Reply:
Yeah, I think we're good. If we find out otherwise, can always fix in a later PR.